mod-audit/src/main/resources/karate-config.js

```diff
@@ -67,10 +67,10 @@ function fn() {
     }
   } else if (env != null && env.match(/^ec2-\d+/)) {
     // Config for FOLIO CI "folio-integration" public ec2- dns name
-    config.baseUrl = 'http://' + env + ':9130';
+    config.baseUrl = 'https://folio-testing-okapi.dev.folio.org:443';
```

Comment:
For mod-audit testing previous value of baseUrl should work, why it was changed to testing endpoint?

Reply:
It does not work for some reason:
![karate_no_diku](https://user-images.githubusercontent.com/83696213/124426798-37148000-dd73-11eb-8c34-a5d268bea225.PNG)